### DISCUSSION TITLE
Custom icon color

### DISCUSSION THREAD

**Discussion by OCram85** [jpanther/congo#60] [Q&A] [answered]
I tried to add some additional fontawesome icons and noticed the color is still black.
Do I have to set the color somehow?

I simply downloaded the fontawesome package and copied the svg icon files into `assets/icons`?

**Comment by jpanther** [comment, marked as answer]
The icons not taking the correct colour from the CSS is usually related to the `fill="currentColor"` being missing from inside the SVG `path` tag. I find when I download the icons from the FontAwesome website directly this is already included, but perhaps if you download their full package it isn't?

Here's a quick screenshot showing the way I normally download them...
![Screen Shot 2021-12-22 at 08 06 27](https://user-images.githubusercontent.com/4462786/146997891-d443d077-c2dc-4bef-822d-9f73d18de1b2.JPG)

**Comment by OCram85** [comment]
Awesome, thanks for the help. I didn't know such svg attributes exists ^^

I hoped, I could avoid downloading single icons  and copied them from the full package.
